core/server/api_container/server/startosis_engine/kurtosis_types/service_config/image_build_spec.go

```diff
@@ -62,6 +64,12 @@ func NewImageBuildSpecType() *kurtosis_type_constructor.KurtosisTypeConstructor
 						return builtin_argument.NonEmptyString(value, TargetStageAttr)
 					},
 				},
+				{
+					Name:              BuildArgsAttr,
+					IsOptional:        true,
+					ZeroValueProvider: builtin_argument.ZeroValueProvider[*starlark.Dict],
+					Validator:         nil,
```

Comment:
Not sure about this :)

Reply:
We can set this to `nil`, we could validate that the dict is empty but that's already handled in the logic so it's fine.